src/kbc_modules/uri.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! Resource Id is the url of KBS URL.
```

Comment:
`//! ResourceId is the identification information of all resources that need to be obtained from `get_resource` endpoint.` is better, I think.